CI/hf_mirror/download_hf_models.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+import yaml
+
+from huggingface_hub import snapshot_download
+from huggingface_hub.utils import RepositoryNotFoundError
+
+
+parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+parser.add_argument('--model_list_file', help='file containing list of models to download.', type=str)
```

Comment:
Is this argument optional? If not, let's remove the `--` prefix or set `required=True`